### DISCUSSION TITLE
Realtime not working (stuck in `'joining'` state?)

### DISCUSSION THREAD

**Discussion by mhchen** [supabase/supabase#8683] [Questions]
Hi, attempting to subscribe to INSERT notifications realtime. I pretty much ran through the entire process that the poster in this thread did to attempt to debug: https://github.com/supabase/supabase/discussions/7861

```
    const chatSubscription = supabase
      .from<ConversationMessage>('chat_messages')
      .on('*', (payload) => { // This was INSERT but is * to debug
        console.log(payload);
      })
      .subscribe();
```

I did notice by inspecting my subscription object that my `state` is stuck in `'joining'`. 

![image](https://user-images.githubusercontent.com/1325549/187055264-5e32953c-7750-4f63-865c-129c3ceea088.png)

A few more pieces of context:
1. Using React but no strict mode
2. RLS off to debug
3. Replication on, Realtime on

<img width="661" alt="image" src="https://user-images.githubusercontent.com/1325549/187055336-1fc75692-ee5a-470d-a0c9-201e04ec53cd.png">

Lastly, I attempted to debug using the SQL statement from that thread `Failed to run sql query: relation "realtime.subscription" does not exist`, but I get this error message, which makes me feel like I'm even more in trouble than the other poster?

<img width="726" alt="image" src="https://user-images.githubusercontent.com/1325549/187055352-17c3aaa3-ba04-40b7-9b75-8c3918d921cb.png">

Is going through the support channel the right recourse? Thanks!

**Comment by GaryAustin1** [comment]
Use the dashboard table UI and select realtime schema and see if there is a subscription table.   If not, and this is hosted system then yes, support is the next stop.  If you are self hosted/docker then you could have setup issues.

**Reply by mhchen** [reply]
Thanks for the reply. It's hosted, and no realtime tables:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/1325549/187058222-810de85c-898c-4b5a-83ca-45e5451404e2.png">

Will open a support ticket. Thanks for confirming!

**Reply by w3b6x9** [reply]
> Will open a support ticket. Thanks for confirming!

@mhchen were you able to get this resolved via support ticket?
